Util/src/main/java/io/deephaven/util/ByteUtils.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+ */
+package io.deephaven.util;
+
+public class ByteUtils {
+    private static final char[] HEX_LOOKUP = new char[] {
+            '0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'a', 'b', 'c', 'd', 'e', 'f'};
+
+    public static String byteArrToHex(byte[] bytes) {
```

Comment:
I feel like this *must* exist in some library we use already.

Reply:
I was using this for reference: https://stackoverflow.com/a/58118078/1028765

The ExecutionContext module is pretty lean. We could add a dependency on guava's impl, but the post says this one is about 4x faster than guavas. DHE is using commons-codec, and that post says (more or less) to avoid that impl (although it is 2-3x faster than the guava impl).

Reply:
We recently moved away from commons-codec elsewhere, and leaned on `io.deephaven.proto.util.ByteHelper#byteBufToHex`. That said, I don't want proto-backplane-grpc depending on Util, nor does it make sense the other way. I think it's okay to have this duplication. (If we really want, we can start a new "byte util" module that has zero dependencies, but that's overkill right now IMO.)